### PR TITLE
Improved code coverage for equivalence cache.

### PR DESCRIPTION
**Pull request by gyliu513**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
none
```

### CODE REVIEW

**Comment by gyliu513** [comment]
@k8s-bot pull-kubernetes-e2e-gce-etcd3 test this

**Comment by gyliu513** [comment]
@k8s-bot pull-kubernetes-e2e-gce-etcd3 test this

**Comment by gyliu513** [comment]
@k8s-bot pull-kubernetes-e2e-gce-etcd3 test this

File: plugin/pkg/scheduler/core/equivalence_cache_test.go [hunk 1]
```diff
@@ -107,18 +185,24 @@ func TestInvalidateCachedPredicateItem(t *testing.T) {
...
-			t.Errorf("Failed : %s, expected invalid: %v, but got: %v", test.name, test.expectedInvalid, invalid)
+		fit, reasons, invalid := ecache.PredicateWithECache(test.pod, test.nodeName, test.predicateKey, test.equivalenceHashForCalPredicate)
+		// returned invalid should match expectedInvalidPredicateKey or expectedInvalidEquivalenceHash
+		if test.equivalenceHashForUpdatePredicate != test.equivalenceHashForCalPredicate {
+			if invalid != test.expectedInvalidEquivalenceHash {
+				t.Errorf("Failed : %s, expected invalid: %v, but got: %v", test.name, test.expectedInvalidEquivalenceHash, invalid)
+			}
+		} else {
+			if invalid != test.expectedInvalidPredicateKey {
+				t.Errorf("Failed : %s, expected invalid: %v, but got: %v", test.name, test.expectedInvalidPredicateKey, invalid)
```
Comment (resouer) [inline plugin/pkg/scheduler/core/equivalence_cache_test.go]: This error message is exactly same with message above. Please add some words to distinguish them.

Comment (gyliu513) [inline plugin/pkg/scheduler/core/equivalence_cache_test.go]: Done

**Comment by resouer** [comment]
@k8s-bot pull-kubernetes-node-e2e test this

**Review by resouer — COMMENTED** [review]
Overall lgtm, let's see if CI is happy

**Review by resouer — APPROVED** [review]
/lgtm

@davidopp any bandwidth to take a look and approve?

**Comment by gyliu513** [comment]
@davidopp can you help approve? Thanks.

**Comment by gyliu513** [comment]
@kubernetes/sig-scheduling-pr-reviews ping for approve.

**Comment by gyliu513** [comment]
ping @jayunit100 @resouer @davidopp for lgtm and approve

**Pull request merged**